server/models/postgis/project.py

```diff
@@ -90,6 +90,9 @@ class Project(db.Model):
     )  # Force users to edit at random to avoid mapping "easy" tasks
     allow_non_beginners = db.Column(db.Boolean, default=False)
     private = db.Column(db.Boolean, default=False)  # Only allowed users can validate
+    featured = db.Column(
+        db.Boolean, default=False
+    )  # Only PMs can set a project as featured
```

Comment:
Current changes lets anyone update a project's `featured` state. Should we include a role check in `set_as_featured` and `unset_as_featured` @JorgeMartinezG

Reply:
Could you please elaborate more on your comment?